fuzzbench/jobs.py

```diff
@@ -19,14 +19,17 @@
 BASE_TAG = 'gcr.io/fuzzbench'
 
 
-def build_image(name: str):
+def build_image(tag: str, context: str, dockerfile=None, buildargs=None):
```

Comment:
consider passing in the object that was returned by `get_images_to_build`

Reply:
OK.
I guess the benefit is we can make the function signature shorter. Also, we can move the logic from run_experiment to here.

Any other benefits?

Reply:
seems cleaner and simpler to me, is there any benefit of the current approach?

Reply:
Done.